.github/workflows/build_cmake.yml

```diff
@@ -50,44 +44,38 @@ jobs:
     - name: Update Submodules
       run: git submodule update --init --recursive --depth=1
 
+    - name: Setup Ninja
+      uses: ashutoshvarma/setup-ninja@v1.1
```

Comment:
@maxxboehme This hasn't released since 2021. Does it install an up-to-date version of Ninja? It uses an outdated version of Node.js and outdated npm dependencies.

Reply:
There is a version [argument](https://github.com/ashutoshvarma/setup-ninja/blob/master/action.yml) that can be past to specify the ninja version. I will look into an alternative way to install ninja onto the build machines.

Reply:
Oops, I didn't notice the `version:` field.

https://github.com/ni/grpc-device/pull/1072/ looks good.